sdccc/src/main/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditions.java

```diff
@@ -74,28 +73,39 @@ private static boolean manipulateMetricStatus(
                 final var resultStartingActivationState =
                         manipulations.setComponentActivation(handle, startingActivationState);
                 // change activation state to something other than expected result to ensure the transition
-                if (resultStartingActivationState != ResponseTypes.Result.RESULT_SUCCESS) {
-                    log.debug(
-                            "Manipulating the activation state to {} was {} for metric state with handle {}."
-                                    + " This needs to be successful, to ensure that the metric is set to {} after performing"
-                                    + " the setMetricStatus manipulation.",
+
+                switch (resultStartingActivationState) {
+                    case RESULT_NOT_SUPPORTED -> log.debug(
```

Comment:
This case is not covered by any unit test.

Reply:
Done